ckanext/validation_schema_generator/plugin/__init__.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+
+import logging
+
+import ckan.plugins as p
+import ckan.plugins.toolkit as tk
+from ckan.exceptions import CkanVersionException
+
+from ckanext.validation_schema_generator.logic.action import _get_actions
+from ckanext.validation_schema_generator.logic.auth import _get_auth_functions
+from ckanext.validation_schema_generator.logic.validators import _get_validators
+from ckanext.validation_schema_generator.helpers import _get_helpers
+
+try:
+    tk.requires_ckan_version("2.9")
```

Comment:
It is cleaner to use `check_ckan_version`, which just returns True/False, than to throw and catch an exception.